db/re/item_db.conf

```diff
@@ -151568,4 +151725,103 @@ item_db: (
 	Loc: 1023
 	Script: <" bonus2 bExpAddRace, RC_All, 2; ">
 },
+{
+	Id: 31016
+	AegisName: "Decorated_Evil_Tree_Card"
+	Name: "Decorated Tree Matilda Card"
+	Type: 6
+	Weight: 10
+	Loc: 16
+	Script: <"
+		bonus bMaxHP, 1000;
+		bonus bMaxSP, 150;
+		bonus bFlee, -25;
+	">
+},
+{
+	Id: 31017
+	AegisName: "Brutal_Tattered_Cookie_Card"
+	Name: "Terrifying Cookie Card"
+	Type: 6
+	Weight: 10
+	Loc: 64
+	Script: <"
+		.@r = getrefine();
```

Comment:
no need to use a variable here check https://github.com/HerculesWS/Hercules/pull/1441